selenium-system-test/src/test/scala/htsstride/browser/Browser.scala

```diff
@@ -121,11 +121,14 @@ trait Assertions { this: WebBrowser with Retrievals with Matchers ⇒
         }
       }
 
+    val errorPrefix: String = if (isError) "Error: " else ""
+    //    val expectedPageTitle = s"$errorPrefix + ${page.expectedPageTitle}"
```

Comment:
The content of the normal and secure Stride journey are slightly different; "eligible for an account" vs. "eligible for a Help to Save account". The content change is currently in a ticket. Once the ticket has been completed, then the page titles should be the same and the tests can pass.

Reply:
the line is commented out - you can probably get rid of it